identity-issuance/src/main/java/com/android/identity/issuance/funke/FunkeIssuingAuthorityState.kt

```diff
@@ -71,20 +71,20 @@ class FunkeIssuingAuthorityState(
                 CredentialFormat.MDOC_MSO -> "funkeMdocMso"
             }
             val issuingAuthorityName = when (credentialFormat) {
-                CredentialFormat.SD_JWT_VC -> "Funke PID Issuer (SD-JWT)"
-                CredentialFormat.MDOC_MSO -> "Funke PID Issuer (MDOC)"
+                CredentialFormat.SD_JWT_VC -> "Funke - SD-JWT"
+                CredentialFormat.MDOC_MSO -> "Funke - mDoc"
```

Comment:
Should this be "SPRIND Funke EUDI Wallet Prototype PID Issuer" instead of just "Funke"? I think there's room enough in the UI for this...

Reply:
It does not fit, but barely. Moved  SD-JWT/mDoc to the description line to fit it.